### DISCUSSION TITLE
strain mapping using MTEX

### DISCUSSION THREAD

**Discussion by Daoud93** [mtex-toolbox/mtex#2039] [Ask Anything]
Does MTEX allow strain mapping if I save EBSD patterns during EBSD mapping? if yes, what type of file formats are needed?. I use TEAM v4.5 software provided by EDAX and it can save the EBSD patterns in the following formats, .PAT, .UP1 or .UP2. Your help will be much appreciated.

**Comment by kilir** [comment]
Dear Daoud,
mtex does not have any build-in functions to determine strain from EBSD pattern. However, in case you have another tool which does that, it might be easy to import the data, plot it, further anylse the data etc. 
Cheers,Rüdiger
